libsplinter/src/admin/store/diesel/operations/add_event.rs

```diff
@@ -0,0 +1,228 @@
+// Copyright 2018-2020 Cargill Incorporated
```

Comment:
Missed 2021 update

Reply:
Updated the copyrights from the added operations files - encompassing your other comments